guides/common/modules/proc_restoring-a-backup-of-a-server-on-rhel8.adoc

```diff
@@ -0,0 +1,10 @@
+[id="Restoring_a_Backup_of_a_Server_on_rhel8_{context}"]
+= Restoring a Backup of a Server on {RHEL} 8
+
+After you perform a fresh installation of {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 instance of the target machine, you can restore the backup you previously created.
+
+.Procedure
+. Update your network configuration to match the target server’s IP address and new host name.
```

Comment:
This is something I commented on in https://github.com/theforeman/foreman-documentation/pull/1311#pullrequestreview-1013052395, but did want to make sure it won't get lost.

1. the IP address doesn't need to match, only the host name
2. the fact that the host name needs to match is already mentioned in the linked "Restoring from Backup" documents below

So please drop the "IP address" part. And I am honestly torn whether the repetition of the other prereq is a good thing or not.

Reply:
I decided to remove the dup.